docs/usage.md

```diff
@@ -117,6 +116,28 @@ When you run the above command, Nextflow automatically pulls the pipeline code f
 nextflow pull qbic-pipelines/rnadeseq
 ```
 
+### Testing the pipeline
+
+A number of test profiles are prepared to allow for easy execution of the pipeline with different parameters to check if these work. Two of these, test_star_rsem and test_star_salmon, do not work by simply running from qbic-pipelines as both require a param pointing to a folder, not a file, and these folders cannot be accessed via raw.githubusercontent.com. Instead you have to either clone the repo, change into the clone and run the pipeline locally, e.g. for the rsem profile, like this:
+
+```bash
+nextflow run . -profile docker,test_star_rsem
+```
+
+Or you have to download the QDESQ folder manually and move it to a folder called testdata, like so:
+
+```bash
+curl https://codeload.github.com/qbic-pipelines/rnadeseq/tar.gz/master | tar -xz --strip=2 rnadeseq-master/testdata/QDESQ
```

Comment:
```suggestion
curl https://codeload.github.com/qbic-pipelines/rnadeseq/tar.gz/master | tar -xz --strip=2 rnadeseq-master/testdata
```

Reply:
Then you don't need to do `mkdir testdata && mv QDESQ testdata`

Reply:
The configs search for paths containing `QDESQ`. QDESQ is saved as a testdata subfolder to keep the different datasets separated; the filenames themselves are the same (i.e. testdata/Sample_preparations.tsv vs testdata/QDESQ/Sample_preparations.tsv).

Reply:
okay well then you can still chain the commands like so 
```suggestion
curl https://codeload.github.com/qbic-pipelines/rnadeseq/tar.gz/master | tar -xz --strip=2 rnadeseq-master/testdata/QDESQ && mkdir testdata && mv QDESQ testdata
```